tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala

```diff
@@ -62,19 +62,16 @@ class GradleLspSuite extends BaseImportSuite("gradle-import") {
     } yield {
       assertNoDiff(
         client.workspaceMessageRequests,
-        List(
-          // Project has .bloop directory so user is asked to "re-import project"
-          importBuildChangesMessage,
-          progressMessage
```

Comment:
How come in this situation and the one down below the `progressMessage` is gone?

Reply:
I noticed this in a lot of places actually.

Reply:
That's because the default response of the `importBuildChanges` is "notNow" (it was "yes" before).
I think it is better to write it explicitly when we want to test a build re-import :
```diff
for {
    ...
+     _ = client.importBuildChanges = ImportBuildChanges.yes
      _ <- server.didSave("build.sc")(identity)
    } yield {
      assertNoDiff(
        client.workspaceMessageRequests,
+       List(
          // Project has .bloop directory so user is asked to "re-import project"
          importBuildChangesMessage,
+         progressMessage
+        ).mkString("\n")
      )
    }
```

Testing that once for each build tool is enough and the rest of the time we just want to test that the user is prompted.
Yet I probably removed too much of them so I am adding some back.

Reply:
Sounds good 👍🏼